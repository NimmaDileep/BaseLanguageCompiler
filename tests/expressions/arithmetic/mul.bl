var x: Int = 5;
var y: Int = 8;

print(x * y);
print(y * x);
