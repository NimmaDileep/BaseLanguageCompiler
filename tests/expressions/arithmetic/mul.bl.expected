40
40
5.55
18.5